src/Data/Nat/DivMod.agda

```diff
@@ -67,6 +75,34 @@ m%n%n≡m%n m (suc n-1) = modₕ-idem 0 m n-1
   (m + n)           % n ≡⟨ [m+n]%n≡m%n m n ⟩
   m                 % n ∎
 
+n≤m⇒[m∸n]%n≡m%n : ∀ {m n} → n ≤ m → .⦃ _ : NonZero n ⦄ →
```

Comment:
Can we move all the `NonZero` arguments further forward in the types, i.e. before `n ≤ m` in this case?

Reply:
Also by convention, we usually introduce the variables in alphabetical order in the name for predictability, e.g. `m≤n⇒[n∸m]%m≡n%m`

Reply:
Ah, I see. Yes. Absolutely.